lib/creator.ex

```diff
@@ -0,0 +1,74 @@
+defmodule WiseHomex.Creator do
+  @moduledoc """
+  Auto creation of main wise_homex module functions from configuration
+
+  These function simply delegate to the api_client() function, which in turn sends a request to either
+  the API client implementation or the mock server
+  """
+
+  @doc """
+  When used by a module, we inject autogenerated mock functions from the api client definition
```

Comment:
The docs in this module have a copy-paste leftover from the mock module.

Reply:
Corrected in https://github.com/wise-home/wise_homex/pull/107/commits/18df924c02db4570a8ffcfde774af6ed65f08751